ArkhamOverlay/Pages/Main/MainView.xaml

```diff
@@ -148,8 +156,9 @@
 
         <StackPanel Orientation="Horizontal" Margin="0, 10, 0, 0">
             <Button Command="{Binding ShowOverlay}">Show Overlay</Button>
-            <Button Margin="10, 0, 0, 0" Command="{Binding ClearCards}">Clear Cards</Button>
-            <Button Margin="10, 0, 0, 0" Command="{Binding ShowAllWindows}">Show All Windows</Button>
+            <Button Margin="10, 0, 0, 0" Command="{Binding ClearCards}" Visibility="{Binding OverlayDisplayed, Converter={StaticResource BoolToVisibility}}">Clear Cards</Button>
+            <Button Margin="10, 0, 0, 0" Command="{Binding ShowAllWindows}" Visibility="{Binding OverlayDisplayed, Converter={StaticResource BoolToVisibility}}">Show All Windows</Button>
```

Comment:
I think this button doesn't need to be tied to the overlay being displayed as the card selection windows can be enabled without the overlay window.